### PR TITLE
make month scroller buttons disabled/enabled and implement logic for …

### PR DESCRIPTION
**Pull request by vipen-made**
…week starts on sunday

### CODE REVIEW

File: calus.js [hunk 1]
```diff
@@ -224,18 +225,26 @@ export default {
...
+
                 if (this.currentDisplayedMonth.month === min.month) {
-                    document.querySelector(prev).classList.add("disabled");
+                    prevEl.classList.add("disabled");
+                    prevEl.disabled = true;
                 } else {
-                    document.querySelector(prev).classList.remove("disabled");
+                    prevEl.classList.remove("disabled");
+                    prevEl.disabled = false;
                 }
```
Comment (lbradley-mm) [inline calus.js]: Wrap in an `if (prevEl)`

File: calus.js [hunk 2]
```diff
@@ -224,18 +225,26 @@ export default {
...
+
                 if (this.currentDisplayedMonth.month === max.month) {
-                    document.querySelector(next).classList.add("disabled");
+                    nextEl.classList.add("disabled");
+                    nextEl.disabled = true;
                 } else {
-                    document.querySelector(next).classList.remove("disabled");
+                    nextEl.classList.remove("disabled");
+                    nextEl.disabled = false;
                 }
```
Comment (lbradley-mm) [inline calus.js]: Same as above, wrap in an `if (nextEl)`

File: calus.js [hunk 3]
```diff
@@ -38,7 +38,8 @@ export default {
     },
     data: function () {
         return {
-            currentDisplayedMonth: DateTime.local().startOf("month")
+            currentDisplayedMonth: DateTime.local().startOf("month"),
+            weekStartOffset: this.weekStartsOnSunday ? -1 : 0,
```
Comment (lbradley-mm) [inline calus.js:42]: Would it be better to pass as a prop?
Could do a start of week as Wednesday if desired that way

**Review by lbradley-mm — CHANGES_REQUESTED** [review]

**Review by mrai-mm — APPROVED** [review]
👍

**Pull request merged**